.github/files/required-review.yaml

```
@@ -7,9 +7,12 @@
    - jetpack-approvers
 
 # Everyone who can approve anything can merge pnpm.lock.
-- name: pnpm lockfile
+- name: pnpm lockfile, composer files
   paths:
    - 'pnpm-lock.yaml'
+   - 'projects/plugins/**/composer.lock'
+   - 'projects/plugins/**/composer.json'
+   - 'projects/plugins/**/changelog/**'
```

Let's be a little more specific here, no need for `**` in these cases.
```suggestion
   - 'projects/plugins/*/composer.lock'
   - 'projects/plugins/*/composer.json'
   - 'projects/plugins/*/changelog/*'
```
The difference being that the `**` version would match like `projects/plugins/foo/bar/baz/changelog/x/y/z` when we really just want to match `projects/plugins/foo/changelog/some-file`.

Same everywhere below where these same rules were added.

Unfortunately there doesn't seem to be a good way to DRY this right now. Yaml supports merging "anchors" into mappings, but not into sequences. So required-reviews would need some code changes, and it's probably not worth holding this PR up on someone (me) doing that.

---

c82c93b